### PR TITLE
Added BreathFirstSearch Algorithm #3392

### PR DESCRIPTION
**Pull request by theSatvik**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Breadth-first search (Uninformed Search)

## Related Issues or Pull Requests

Artificial Intelligence Algorithms #3392

### CODE REVIEW

**Comment by theSatvik** [comment]
Hi @siddharth25pandey, I have added the following algorithm passed the check-in python also. Thankyou.

File: AI-Algorithms/breathFirstSearch.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+graph.add_edge('A', 'B')
+graph.add_edge('A', 'S')
+graph.add_edge('S', 'G')
+graph.add_edge('S', 'C')
+graph.add_edge('C', 'F')
+graph.add_edge('G', 'F')
+graph.add_edge('C', 'D')
+graph.add_edge('C', 'E')
+graph.add_edge('E', 'H')
+graph.add_edge('G', 'H')
```
Comment (siddharth25pandey) [inline AI-Algorithms/breathFirstSearch.py:73]: Take dynamic input from the user

File: AI-Algorithms/breathFirstSearch.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+start, goal = 'A', 'H'
```
Comment (siddharth25pandey) [inline AI-Algorithms/breathFirstSearch.py:74]: Take dynamic input from the user

File: AI-Algorithms/breathFirstSearch.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+traced_path = graph.breadth_first_search(start, goal)
+
+if traced_path:
+    print('Path:', end=' ')
```
Comment (siddharth25pandey) [inline AI-Algorithms/breathFirstSearch.py:78]: Display a message in print form as what are you showing output to user for better user-interface

File: AI-Algorithms/breathFirstSearch.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    Graph.print_path(traced_path, goal)
+    print()
```
Comment (siddharth25pandey) [inline AI-Algorithms/breathFirstSearch.py:80]: At the end, write Sample Input & Output and write the Time and Space Complexity of the Algorithm.
Do the same for all PR you gonna create in future

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@theSatvik Add a Readme file and do the changes as suggested below

**Comment by siddharth25pandey** [comment]
@theSatvik Also add the newly created folder in the Python Directory

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top

@theSatvik Do the changes mentioned by @siddharth25pandey as well.

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this PR due to inactivity.

**Pull request closed without merging**